cluster/pre-existing/util.sh

```diff
@@ -60,3 +60,18 @@ function create-certs {
   KUBEAPISERVER_CERT_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
   KUBEAPISERVER_KEY_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
 }
+function detect-master() {
+  detect-project
+  KUBE_MASTER=${MASTER_NAME}
+  echo "Trying to find master named '${MASTER_NAME}'" >&2
+  if [[ -z "${KUBE_MASTER_IP-}" ]]; then
+    local master_address_name="${MASTER_NAME}-ip"
+    echo "Looking for address '${master_address_name}'" >&2
+    KUBE_MASTER_IP=${MASTER_IP}
```

Comment:
where does `$MASTER_IP` get set? The line before echos that you are looking for `$master_address_name` but I don't see anything that actually looks for it.

Reply:
You can refer to the following link https://github.com/kubernetes/kubernetes/tree/master/test/kubemark/pre-existing
It introduced how to change the related configuration file and it provided an example at the end.

Reply:
/cc @gmarek - can you verify that this looks correct?

Reply:
@shyamjvs is current owner